exercises/resistor-color-trio/description.md

```diff
@@ -23,7 +23,7 @@ For this exercise, you need to know only three things about them:
 - Grey: 8
 - White: 9
 
-In `resistor-color duo` you decoded the first two colors.
+In `Resistor Color Duo` you decoded the first two colors.
```

Comment:
Is the preservation of the backticks intentional?

Reply:
It seemed their addition before was intentional (maybe draw attention?) so I left it alone. I don't have any strong feelings on whether to keep the backticks or not.

Reply:
I think they can be removed here. The previous version was a slug, which we regularly do use backticks on, but not for exercise titles I think.

```suggestion
In Resistor Color Duo you decoded the first two colors.
```

Reply:
+1. Backticks are used around syntax, code, filenames and other words that are not "regular" prose/English. They should not be used around an exercise title.